templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.SQL/backend/sql/sql_client.py

```diff
@@ -1,44 +1,23 @@
-import azure.cosmos.cosmos_client as cosmos_client
+from azure.cosmos import CosmosClient, errors, http_constants, PartitionKey
 import azure.cosmos.errors as errors
 import azure.cosmos.http_constants as http_constants
```

Comment:
```suggestion
```

Reply:
in fact... none of those are used anymore? should we still need to add any **try** **except** ?